src/components/dialog.css

```diff
@@ -90,3 +90,16 @@
     margin-left: 0 !important;
   }
 }
+
+/**
+ * Class applied to content page to disable scrolling.
+ */
+body.swg-disable-scroll {
+  margin: 0 !important;
```

Comment:
Wondering if the margin 0 might screw up some sites (I know I suggested it but I'm not 100% sure it's needed to stop scrolling)

Reply:
I'm also not sure if that is needed. I can remove it, to err on the side of being less intrusive.

Reply:
Probably a good idea, we can always add it if needed later.